pkg/yurttunnel/agent/agent.go

```diff
@@ -35,6 +35,34 @@ import (
 	"github.com/alibaba/openyurt/pkg/yurttunnel/pki/certmanager"
 )
 
+// TunnelAgent sets up tunnel to TunnelServer, receive requests
+// from tunnel, and forwards requests to kubelet
+type TunnelAgent interface {
```

Comment:
now we make up interface for tunnel agent and tunnel server, it is better to separate the interface implementation(for example: anp tunnel) into independent dir or file.

Reply:
create independent files for them